src/test/java/com/karankumar/bookproject/backend/entity/UserTest.java

```diff
@@ -27,7 +27,7 @@ void beforeEachSetup() {
     @Test
     void testValidUserSaved() {
         // given
-        User user = userWithoutPassword().password("passwordP1&")
+        User user = userWithoutPassword().password("passwordP1&").email("abc@def.com")
```

Comment:
Could you drop the `.email` method invocation down one line (aligned with `.password`)?

Reply:
Yes, I fixed it